### PR TITLE
Allow camera live feed in front.

### PR DESCRIPTION
**Pull request by hanwu85**
allow the live feed screen to face the compeititors.

### CODE REVIEW

File: wca-regulations.md [hunk 1]
```diff
@@ -88,7 +88,7 @@ Note: Because Article and Regulation numbers are not reassigned when Regulations
         - 2i1c) Hand warmers.
         - 2i1d) Food and drink.
     - 2i2) Competitors may use cameras at the solving station at the discretion of the WCA Delegate, but the following restrictions apply from the start of the attempt until the competitor stops the solve. Penalty for breaking a restriction: disqualification of the attempt (DNF).
-        - 2i2a) Each camera monitor must be blank or out of sight of the competitor (see [Regulation A5b](regulations:regulation:A5b)).
+        - 2i2a) The camera monitor may show the live feed to the competitors as long as the competitors do not gain information from it during the solve (see [Regulation A5b](regulations:regulation:A5b)).
```
Comment (hanwu85) [inline wca-regulations.md]: differnt option with inspection added.
```suggestion
        - 2i2a) The camera monitor may show the live feed to the competitors as long as the competitors do not gain information from it during the solve and the inspection(see [Regulation A5b](regulations:regulation:A5b)).
```

Comment (ohexter) [reply wca-regulations.md]: What do we mean by gaining information and how will we assess this?

Comment (ohexter) [inline wca-regulations.md]:
```suggestion
        - 2i2a) Each camera monitor must be blank or out of sight of the competitor. Exception: if a visible active monitor is discovered, completed attempts need not be disqualified at the discretion of the WCA Delegate if the competitor did not receive assistance from the monitor (see [Regulation A5b](regulations:regulation:A5b)), however the competitor must not be permitted to perform any further attempts with such a monitor. 
```

My alternate suggestion

File: wca-regulations.md [hunk 2]
```diff
@@ -88,7 +88,7 @@ Note: Because Article and Regulation numbers are not reassigned when Regulations
         - 2i1c) Hand warmers.
         - 2i1d) Food and drink.
     - 2i2) Competitors may use cameras at the solving station at the discretion of the WCA Delegate, but the following restrictions apply from the start of the attempt until the competitor stops the solve. Penalty for breaking a restriction: disqualification of the attempt (DNF).
-        - 2i2a) Each camera monitor must be blank or out of sight of the competitor (see [Regulation A5b](regulations:regulation:A5b)).
+        - 2i2a) Each camera monitor must be blank or out of sight of the competitor. Exception: if a visible active monitor is discovered, completed attempts need not be disqualified at the discretion of the WCA Delegate if the competitor did not receive assistance from the monitor (see [Regulation A5b](regulations:regulation:A5b)), however the competitor must not be permitted to perform any further attempts with such a monitor.
```
Comment (CarterKoala) [inline wca-regulations.md]:
```suggestion
        - 2i2a) Each camera monitor must be blank or out of sight of the competitor. Exception: if a visible active monitor is discovered, completed attempts need not be disqualified if the competitor did not receive assistance from the monitor at the discretion of the WCA Delegate (see [Regulation A5b](regulations:regulation:A5b)), however the competitor must not be permitted to perform any further attempts with such a monitor. 
```
Minor Wording change suggestion

Comment (ohexter) [reply wca-regulations.md]:
```suggestion
       - 2i2a) Each camera monitor must be blank or out of sight of the competitor. Exception: if a visible active monitor is discovered, completed attempts need not be disqualified if the competitor did not receive assistance from the monitor, at the discretion of the WCA Delegate (see [Regulation A5b](regulations:regulation:A5b)), however the competitor must not be permitted to perform any further attempts with such a monitor. 
```
I think we need an extra comma if we're doing it this way

**Pull request merged**